### PR TITLE
[WIP] docs(repackaging): repackage sample to the new world

### PR DESCRIPTION
**Pull request by wardbell**
**WIP DO NOT MERGE**

### CODE REVIEW

File: public/docs/_examples/cb-ts-to-js/js/app/main.js [hunk 1]
```diff
@@ -7,7 +7,7 @@
   var provide =
     ng.core.provide;
   var bootstrap =
-    ng.platform.browser.bootstrap;
+    ng.platformBrowserDynamic.bootstrap;
   var LocationStrategy =
     ng.router.LocationStrategy;
```
Comment (ericmartinezr) [inline public/docs/_examples/cb-ts-to-js/js/app/main.js]: ng.platform.common.LocationStrategy

The same for HashLocationStrategy

Comment (filipesilva) [inline public/docs/_examples/cb-ts-to-js/js/app/main.js]: @ericmartinezr I'm including that change in a commit now.

**Comment by naomiblack** [comment]
this seems to introduce a Harp error in the Dart version of the glossary http://localhost:3000/docs/dart/latest/guide/glossary.html

```
TypeError - /Users/naomib/github/angular.io/public/docs/dart/latest/guide/glossary.jade:1 > 1| != partial("../glossary") 2| /Users/naomib/github/angular.io/public/docs/dart/latest/glossary.jade:7 5| !=partial('../../ts/latest/_fragments/glossary-intro') 6| !=partial('../../ts/latest/_fragments/glossary-a-2') > 7| !=partial('../../ts/latest/_fragments/glossary-b-c') 8| !=partial('../../ts/latest/_fragments/glossary-d1') 9| !=partial('../../ts/latest/_fragments/glossary-d2') 10| !=partial('../../ts/latest/_fragments/glossary-e1') Cannot read property 'call' of undefined
```

**Comment by naomiblack** [comment]
merged and live.

**Comment by wardbell** [comment]
Closed by d45c7e26f6fdf29d11fa0a3c8ae59651032d377c

**Pull request closed without merging**